src/test/kotlin/calculator/StringAddCalculatorTest.kt

```diff
@@ -0,0 +1,61 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringAddCalculatorTest {
+
+    private lateinit var calculator: StringAddCalculator
+
+    @BeforeEach
+    fun setUp() {
+        calculator = StringAddCalculator();
```

Comment:
지금과 같은 케이스라면 `@BeforeEach` 없이 아래처럼 선언도 가능합니다.

```suggestion
    private val calculator = StringAddCalculator();
```